crates/test-utils/Cargo.toml

```diff
@@ -33,8 +33,8 @@ rstest = { workspace = true }
 git2 = { workspace = true, optional = true }
 
 futures = "0.3.26"
-serde = { version = "1.0" }
-serde_json = { version = "1.0", features = ["preserve_order"] }
+serde = { workspace = true}
```

Comment:
Just fix the formatting issue, then lgtm 
workspace = true }

Reply:
Ah, interesting, cargo fmt should also do that! Don't know why it doesn't.

Reply:
Done [here](https://github.com/kkrt-labs/kakarot-rpc/pull/553/commits/773ff312844dc8d7be2e7b7f986d0a8a6091d027).